src/coreclr/debug/shared/dbgtransportsession.cpp

```diff
@@ -2203,6 +2203,7 @@ DWORD DbgTransportSession::GetEventSize(DebuggerIPCEvent *pEvent)
     case DB_IPCE_AFTER_GARBAGE_COLLECTION:
     case DB_IPCE_DISABLE_OPTS_RESULT:
     case DB_IPCE_CATCH_HANDLER_FOUND_RESULT:
+    case DB_IPCE_SET_ENABLE_CUSTOM_NOTIFICATION_RESULT:    
```

Comment:
Is this intentionally part of this PR?

Reply:
This was causing a boring assertion in the debug build, I can open a separate PR with this fix but it can be here together.

Reply:
I don't have a strong preference.  Would we want to include that changes as part of a backport to .NET servicing?

Reply:
Ah, okay, I will remove it, it will be better for backporting only the change.